android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -801,10 +801,23 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
 
     @Override
     public void onReceivedSslError(final WebView webView, final SslErrorHandler handler, final SslError error) {
+        // WebView.getUrl() will return the top-level window URL.
+        // If a top-level navigation triggers this error handler, the top-level URL will be the failing URL (not the current URL).
```

Comment:
"current" is slightly confusing in this transitional context. I assume it means "last successful URL." If you initialize a WebView with a URL that has invalid SSL cert, what is the "current" URL? Just blank?

Reply:
"current" is meant to say "currently rendered page", which could be a blank page if none is currently rendered. Will update shortly.